### PR TITLE
Add Gothic Mod Build Tool project

### PR DESCRIPTION
**Pull request by Szmyk**

### CODE REVIEW

File: _data/projects/gmbt.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+tags:
+- ".NET"
+- C#
+- modding
+- gamedev
+
+upforgrabs:
+  name: easy
+  link: https://github.com/Szmyk/gmbt/labels/easy
```
Comment (ritwik12) [inline _data/projects/gmbt.yml:13]: @Szmyk I can't see any issues labeled with "easy".

Comment (ritwik12) [reply _data/projects/gmbt.yml:13]: Also please remove extra lines.

**Comment by shiftkey** [comment]
@Szmyk looks good, but I'll hold off on merging this until there are some issues listed with the associated label...

**Comment by Szmyk** [comment]
OK, thanks! I will @ you, if there will be some issues to resolve :)

File: _data/projects/gmbt.yml (hunk 1 above)
Comment (shiftkey) [inline _data/projects/gmbt.yml]: This all seems fine, and is ready to merge when there are issues created and assigned to this label...

**Comment by shiftkey** [comment]
Closing this out due to inactivity.

@Szmyk please open a fresh PR if you would still like the project to be listed.

**Pull request closed without merging**